src/Console/SshCommand.php

```diff
@@ -49,7 +49,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $host = $this->deployer->hosts->get($input->getArgument('hostname'));
+        $firstHostname = $this->deployer->hosts->getIterator()->current()->getHostname();
```

Comment:
what I meant: this fallback to the first hostname only makes sense where there is just one defined.

using the first when several are defined doesnt sound like a usefull guess.